core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -462,13 +462,14 @@ protected double parseSpeed( String str )
     }
 
     /**
-     * This method parses a string ala "00:00" (hours and minutes) or "0:00:00" (days, hours and
-     * minutes).
+     * This method parses a string ala 'hh:mm', format for hours and minutes 'mm', 'hh:mm' or 'hh:mm:ss'
+     * FIXME: Add support for ISO_8601 
```

Comment:
Maybe we can log these mismatches and create a blog post with these entries so that we or mappers could fix them